packages/wagmi-core/src/actions/sendEmailOtp.ts

```diff
@@ -1,5 +1,5 @@
-import type { Config } from "wagmi";
 import { resolveAlchemyAuthConnector } from "@alchemy/connectors-web";
+import type { Config } from "@wagmi/core";
```

Comment:
consolidating all imports on `@wagmi/core` to avoid type errors with `wagmi` import